app/javascript/redux/actions.js

```diff
@@ -0,0 +1,14 @@
+export const FETCH_DATA_REQUEST = 'FETCH_DATA_REQUEST';
+export const FETCH_DATA_SUCCESS = 'FETCH_DATA_SUCCESS';
+export const FETCH_DATA_FAILURE = 'FETCH_DATA_FAILURE';
+
+export const fetchData = () => async (dispatch) => {
+    dispatch({ type: FETCH_DATA_REQUEST });
+    try {
+      const response = await fetch('https://rails-hello-back-end.onrender.com/');
+      const data = await response.json();
```

Comment:
@Omar-Muhamad I though the instruction said I should fetch data from another another API endpoint, which exactly what I have done here. You may please clarify what exactly you mean by "Add seeds for the greetings and fetch data from the API endpoint"

Reply:
* As mentioned by the previous reviewer you are to 

> Implement a connection between a Ruby on Rails back-end and React front-end but differently as the name of the milestone implies `Set up a 'Hello world' project as one app with Webpack`. that is your API should be your rails app, not another External API, that has a Table created for storing your messages( the messages the API would send to your React app) and creating 5 different greetings( you can achieve this by adding greetings through seeding [FIND more details here](https://ninjadevel.com/seeding-database-ruby-on-rails/)) after which you will set up your react side with actions, and reducers used to fetch from the API endpoint you created above in your rails app ( ie the endpoint can look like this `http://localhost:3000/api/greetings` ) and by running `./bin/dev ` webpacker bundles your react app and renders it. please go through the lessons provided for this milestone so that you can understand the learning objectives 

> Understand the pros and cons of different approaches of connecting Ruby on Rails back-end with React front-end.